tests/console/verify_systemd_timesync.pm

```diff
@@ -0,0 +1,48 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify that systemd timer is used for time synchronization.
+# Maintainer: Sofia Syrianidou <ssyrianidou@suse.com>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub run {
+    my $test_data = get_test_suite_data();
+    select_console 'root-console';
+    record_info("Check yast-timesync.timer", "Check if file /etc/systemd/system/yast-timesync.timer exists and has expected configuration.");
```

Comment:
One thing that is used and help to organize the code is to separate blocks of actions in the test and even document them in the description of the test on the top, so for example one blank line between actions helps or even if that block is preceded with some `record_info`. In this test you are doing something like (1) check conf files (2) check first sync in the logs (3) add deviation fix, (4) check next sync in the logs.
This is completely a matter of taste :) , so feel free to not act on this if you prefer to have it more compact or in any particular way.

Reply:
Done